tck/src/main/java/org/eclipse/microprofile/lra/tck/TckLRATypeTests.java

```diff
@@ -286,10 +311,14 @@ private void resourceRequest(String path, boolean startLRA, int expectedStatus,
             if (lra != null) {
                 lraClient.closeLRA(lra);
             }
+            
+            return methodLRA;
```

Comment:
It somewhat contravenes my other comment but how about changing this to the return incomingLRA (I assume this is an ID) and only return it if you call lraClient.closeLRA(lra); (i.e move it a few lines up).

I am not sure if you agree but I think it might help readability of the new tests.

Reply:
I can change it to incomingLRA, they are the same just methodLRA is returned from the resource in the response body. But I cannot change to return it only when the lra is closed (I guess you refer to [this](https://github.com/eclipse/microprofile-lra/pull/192/files#diff-33b883a0ff0f67ccf92bc5322022c6cfR305)) because we want to return it even in case the LRA was closed when the REST method finished (automatically). Sorry if I am missing what you meant.

Reply:
I was thinking about [this](https://github.com/eclipse/microprofile-lra/pull/192/files#diff-33b883a0ff0f67ccf92bc5322022c6cfR312) but I can appreciate your comment here. I think the main thing is to make it clear what is being returned (I am thinking from the point of view of the review of this PR I could not immediately decipher what might be in "methodLRA" and was thinking it would only be required in that case I link to earlier in this comment but I could be wrong there.

perhaps
> return methodLRA; // expected to contain the LRA ID

Would be enough?

Reply:
ah, now I understand :) thanks, will update this all in one go later today